src/core/control/Control.cpp

```diff
@@ -65,6 +66,7 @@
 #include "gui/toolbarMenubar/ToolMenuHandler.h"                  // for Tool...
 #include "gui/toolbarMenubar/model/ToolbarData.h"                // for Tool...
 #include "gui/toolbarMenubar/model/ToolbarModel.h"               // for Tool...
+#include "model/BackgroundImage.h"                               // for pngOpen
```

Comment:
You use it for `BackgroundImage` in the first place.
```suggestion
#include "model/BackgroundImage.h"                               // for Back...
```